Index: webalizer-2.01.10/Makefile.in
===================================================================
--- webalizer-2.01.10.orig/Makefile.in	2006-10-28 07:06:46.000000000 +0400
+++ webalizer-2.01.10/Makefile.in	2006-10-28 07:06:58.000000000 +0400
@@ -47,7 +47,7 @@
	$(CC) ${LDFLAGS} -o webalizer webalizer.o hashtab.o linklist.o \
		preserve.o parser.o output.o dns_resolv.o graphs.o \
		gdfontg.o gdfontl.o gdfontmb.o gdfonts.o gdfontt.o \
-		${LIBS}
+		-Wl,-Bstatic -lGeoIP -lgd -lpng -ldb -Wl,-Bdynamic -lz -lm
	rm -f webazolver
	@LN_S@ webalizer webazolver

@@ -88,16 +88,17 @@
	@LN_S@ lang/webalizer_lang.@DEFAULT_LANG@ webalizer_lang.h

 install: all
-	$(INSTALL_PROGRAM) webalizer ${BINDIR}/webalizer
-	$(INSTALL_DATA) webalizer.1 ${MANDIR}/webalizer.1
-	$(INSTALL_DATA) sample.conf ${ETCDIR}/webalizer.conf.sample
-	rm -f ${BINDIR}/webazolver
-	@LN_S@ ${BINDIR}/webalizer ${BINDIR}/webazolver
+	$(INSTALL_PROGRAM) webalizer ${DESTDIR}/${BINDIR}/webalizer
+	$(INSTALL_DATA) webalizer.1 ${DESTDIR}/${MANDIR}/webalizer.1
+	$(INSTALL_DATA) sample.conf ${DESTDIR}/${ETCDIR}/webalizer.conf.sample
+	for a in $(LANGS); do $(INSTALL_DATA) po/$$a.mo ${DESTDIR}/$(LOCALEDIR)/$$a/LC_MESSAGES/webalizer.mo; done
+	rm -f ${DESTDIR}/${BINDIR}/webazolver
+	(cd ${DESTDIR}/${BINDIR}; @LN_S@ webalizer webazolver)

 uninstall:
-	rm -f ${BINDIR}/webalizer
-	rm -f ${BINDIR}/webazolver
-	rm -f ${MANDIR}/webalizer.1
-	rm -f ${ETCDIR}/webalizer.conf.sample
+	rm -f ${DESTDIR}/${BINDIR}/webalizer
+	rm -f ${DESTDIR}/${BINDIR}/webazolver
+	rm -f ${DESTDIR}/${MANDIR}/webalizer.1
+	rm -f ${DESTDIR}/${ETCDIR}/webalizer.conf.sample
	rm -f webalizer_lang.h
	@LN_S@ lang/webalizer_lang.@DEFAULT_LANG@ webalizer_lang.h
